packages/sanity/test/form/index.ts

```diff
@@ -1,4 +1,3 @@
-export * from './renderArrayOfObjectsInput'
```

Comment:
Just to double check, this is intentional correct?

Reply:
I'm just trying to understand why or perhaps it was implemented somewhere else?

Reply:
Good catch. Now I'm not so sure. It was not in use, so it's safe, but not sure if either me or Binoy intended to delete it. Added it back – we can always delete it again later.